RTN-011.tex

```diff
@@ -143,12 +145,13 @@
   \addtohist{19}{2025-02-10}{Major updates to the plans and data products for DP1. Revision of the Alert Production section. Release version 6.0}{Leanne Guy}
   \addtohist{20}{2025-02-24}{Minor updates to add more information on DP1 data products. Release version 6.1}{Leanne Guy}
   \addtohist{21}{2025-05-30}{Updates to DP1 data products, release timeline and moving object alert packet, Release version 6.2}{Melissa Graham, Leanne Guy, Eric Bellm}
-  \addtohist{22}{2025-08-19}{Create Rubin DOI}{T. Jenness}
+    \addtohist{21}{2025-11-07}{Updates to LSSTCam commissioning  release timeline.}{Leanne Guy, Eric Bellm, Lynne Jones. }
```

Comment:
Something's gone wrong here, with "21" reused and @timj's entry lost

Reply:
Also presumably we need to add "Release version 6.3" or 7.0, as appropriate.

Reply:
Next ...